filters/crop/CropFilter.cpp

```diff
@@ -154,6 +154,14 @@ PointViewSet CropFilter::run(PointViewPtr view)
         crop(box.to2d(), *view, *outView);
         viewSet.insert(outView);
     }
+
+    for (auto& point: m_points)
```

Comment:
I will rename. They can't be merged into geoms because they aren't the same thing.